_docs/principles/accessibility.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Accessibility
+layout: docs
+category: Principles
+usage: >
+  Visual accessibility standards enable users with varied levels of color impairment to successfully navigate and use a Helix based UI.
+status: Stable
+resource: true
+last-modified: 2017-10-24
+---
+
+
+## Accessibility for visual design
+
+<div class="hxRow">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+Rackspace products must be made accessible for everyone – regardless of a person’s abilities. We can assume from the statistics that many of our users have some form of visual impairment. [Colour Blind Awareness](http://www.colourblindawareness.org/) estimates that: 
+
+- 4.5% of the global population experience color blindness (1 in 12 men and 1 in 200 women)
+- 4% suffer from low vision (1 in 30 people)
+- 0.6% are blind (1 in 188 people)
+ 
+By using the following best practices, you can help ensure your design is accessible for the visually impaired.
+
+### Do not rely on color alone to convey meaning
```

Comment:
Should we wrap the "Do not rely on..." section in helix row markup to gain the benefit of some vertical spacing between it and the section above?
Not sure if there's a better solution, other than to create a `<section class="website-section">` and add some space.  Any thoughts @nimbinatus ?

Reply:
Wrapped in `hxRow`